task/monorail-server/src/bsp/sidecar_bcd.rs

```diff
@@ -16,8 +16,7 @@ task_slot!(SEQ, seq);
 task_slot!(FRONT_IO, ecp5_front_io);
 
 /// Interval at which `Bsp::wake()` is called by the main loop
-const WAKE_INTERVAL_MS: u64 = 500;
-pub const WAKE_INTERVAL: Option<u64> = Some(WAKE_INTERVAL_MS);
+pub const WAKE_INTERVAL: Option<u32> = Some(500);
```

Comment:
**thought:** Probably a good reason for this somewhere below but an optional Some interval is an interesting thing to see :)